modules/story-budget/story-budget.php

```diff
@@ -388,9 +388,17 @@ function get_posts_for_term( $term, $args = null ) {
 		// Filter for an end user to implement any of their own query args
      	$args = apply_filters( 'ef_story_budget_posts_query_args', $args );
```

Comment:
Would moving the filter to be right before the WP_Query be more useful? That way the new parameters we're adding an be filtered out.
I'd also make it clear somewhere in the PR's description that this could potentially be a breaking change. We should note it in the Release Notes when we release a new version.

Reply:
I missed that filter! I'm going to move it around.